reference/docs-conceptual/how-to-use-docs.md

```diff
@@ -0,0 +1,35 @@
+---
+ms.date:  09/25/2019
+keywords:  powershell,cmdlet
+title:  How to use the PowerShell documentation
+---
+# How to use the PowerShell documentation
+
+Welcome to the PowerShell online documentation. This site contains cmdlet reference for the
+following versions of PowerShell:
+
+- PowerShell 7 (preview)
+- PowerShell 6
+- PowerShell 5.1
+- PowerShell 5.0
+- PowerShell 4.0
+- PowerShell 3.0
+
+## Selecting your version
+
+By default, this site displays documentation for the latest released version of PowerShell. Some
+cmdlets work differently in various versions of PowerShell. Be sure you are viewing the
+documentation for the version of PowerShell you are using.
+
+Use the version picker at the top of the page to select the version of PowerShell you want.
+
+![version picker](images/how-to-use-docs/picker-vall.gif)
+
+## Searching for articles
+
+There are two ways to search for content in Docs. The simplest way is to use the filter box under
+the version selector. Just enter a word that appears in the title of an article. The page displays
+a list of matching articles. You can also select the option to search the entire site from that
+list.
+
+![filter box](images/how-to-use-docs/filter-search.gif)
```

Comment:
Should this have next steps or additional info, like maybe links to contributing or something like that?  It doesn't feel like it leads into anything.

Reply:
Not for now.